MapboxDirections/MBAttribute.h

```diff
@@ -0,0 +1,10 @@
+typedef NS_OPTIONS(NSUInteger, MBAttribute) {
```

Comment:
One more rename! Now that this is an option set instead of a single attribute, rename this type to MBAttributeOptions, so it’s easier to talk about individual attributes versus the whole set of attributes.